internal/pkg/template/templates/environment/cf.yml

```diff
@@ -146,6 +179,20 @@ Resources:
       Subnets: [ {{range $id := .VPCConfig.Imported.PublicSubnetIDs}}{{$id}}, {{end}} ]
 {{- else}}
       Subnets: [ {{range $ind, $cidr := .VPCConfig.Managed.PublicSubnetCIDRs}}!Ref PublicSubnet{{inc $ind}}, {{end}} ]
+{{- end}}
+      Type: application
+  InternalLoadBalancer:
```

Comment:
What do you think of writing it such that resources related to internal LB are close together? Would it be easier in the future to navigate? This LGTM as well though!!